static/templates/confirm_unstar_all_messages_in_topic.hbs

```diff
@@ -1,5 +1,5 @@
 <p>
     {{#tr this}}
-    Are you sure you want to unstar all messages in topic <b>__topic__</b>?  This action cannot be undone.
+    Are you sure you want to unstar all messages in <strong><span class="stream">__stream_name__</span> > <span class="topic">__topic__</span></strong>?  This action cannot be undone.
```

Comment:
I think we don't want all this HTML syntax to be in what we ask translators to translate; ideally we'd just have a `__stream_topic__` variable that sits in the translatable string.  I'm not sure how convenient this is to do with` tr this`; here's our comment on its definition from `templates.js`:

```
Handlebars.registerHelper("tr", (context, options) => {                                              
    // Marks a block for translation.                                                                
    // Example usage 1:                                                                              
    //     {{#tr context}}                                                                           
    //         <p>some English text</p>                                                              
    //     {{/tr}}                                                                                   
    //                                                                                               
    // Example usage 2:                                                                              
    //     {{#tr context}}                                                                           
    //         <p>This __variable__ will get value from context</p>                                  
    //     {{/tr}}                                                                                   
    //                                                                                               
    // Notes:                                                                                        
    //     1. `context` is very important. It can be `this` or an                                    
    //        object or key of the current context.                                                  
    //     2. Use `__` instead of `{{` and `}}` to declare expressions                               
```

@hackerkid may be able to help if you need it; it's possible what we end up doing is making a template that we pass the stream/topic into, and passing the rendered version of that into this template.

Reply:
I tried the sperate template method locally and it seems to work, but I couldn't figure out if the text gets translated correctly, since the string is new and translation data isn't available.
Opened https://chat.zulip.org/#narrow/stream/49-development-help/topic/HTML.20inside.20translation.20tags.2E for this.

Reply:
This PR now uses the new syntax introduced in #17999, so there's fewer HTML inside the translation tags.